native/opencv-gpu.cc

```diff
@@ -177,102 +151,83 @@ Size2i cv_gpu_hog_get_win_stride(GpuHog* hog) {
 // =============================================================================
 //   CascadeClassifier
 // =============================================================================
-using GpuCascadePtr = cv::Ptr<cv::cuda::CascadeClassifier>;
-
-GpuCascade* cv_gpu_cascade_new(const char* const filename) {
-    auto cascade = cv::cuda::CascadeClassifier::create(filename);
-    return reinterpret_cast<GpuCascade*>(new GpuCascadePtr(cascade));
+void* cv_gpu_cascade_new(const char* const filename) {
+    return cv::cuda::CascadeClassifier::create(filename);
```

Comment:
Missing a new here, i.e. `new cv::Ptr<cv::cuda::CascadeClassifier>`

Reply:
Yes, I see.

Why travis didn't handle it?